src/main/java/by/issoft/client/ZipCodeClient.java

```diff
@@ -0,0 +1,51 @@
+package by.issoft.client;
+
+import by.issoft.ResponseEntity;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpResponse;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class ZipCodeClient {
+
+    private static final String GET_ZIPCODES_ENDPOINT = "/zip-codes";
+    private static final String POST_ZIPCODES_EXPAND_ENDPOINT = "/zip-codes/expand";
+    private final ObjectMapper objectMapper;
+
+    public ZipCodeClient() {
+        this.objectMapper = new ObjectMapper();
+    }
+
+    public ResponseEntity<List<String>> getZipCodes() {
+        List<String> zipCodes;
+        ResponseEntity<List<String>> response = new ResponseEntity<>();
+        HttpResponse httpResponse = Client.doGet(GET_ZIPCODES_ENDPOINT);
+        response.setStatusCode(httpResponse.getStatusLine().getStatusCode());
+        try {
+            zipCodes = Arrays.stream(objectMapper.readValue(httpResponse.getEntity().getContent(), String[].class)).toList();
+            response.setBody(zipCodes);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return response;
+    }
+
+    public ResponseEntity<List<String>> postZipCodes(String... zipcodes) {
+        List<String> zipCodes;
+        ResponseEntity<List<String>> response = new ResponseEntity<>();
+        for (int i=0; i< zipcodes.length; i++) {
```

Comment:
the endpoint provides to post list of zipcodes. It is not good to use requests in a loop. Use 
`HttpResponse httpResponse = Client.doPost(POST_ZIPCODES_EXPAND_ENDPOINT, Arrays.toString(zipcodes));`
`Arrays.toString(zipcodes));` will create an entry in the format you need [{zip}, {zip}, {zip}, ...]